site/content/contribute/mobile/developer-setup/_index.md

```diff
@@ -40,7 +45,10 @@ Other installation options:
 -   Using NVM by following the instructions [here](https://github.com/creationix/nvm#install-script).
 -   Download and install the package from the [NodeJS website](https://nodejs.org/en).
 
-#### Install [Watchman](https://facebook.github.io/watchman
+##### Windows 10
+-   Download and install the package from the [NodeJS website](https://nodejs.org/en/)
```

Comment:
@enahum line 46 refers to Linux specifically
![image](https://user-images.githubusercontent.com/13119842/96712259-d7a14c00-136c-11eb-9fc0-aa8f9c86aca0.png)

Perhaps for a later iteration we can consider restructuring this page similar to server developer set up, so that it uses separate tabs for the different OS's
![image](https://user-images.githubusercontent.com/13119842/96712287-e5ef6800-136c-11eb-9010-7b973739d7fe.png)